token/services/ttx/endorse.go

```diff
@@ -367,6 +363,34 @@ func (c *collectEndorsementsView) requestApproval(context view.Context) (*networ
 	return env, nil
 }
 
+func (c *collectEndorsementsView) requestAudit(context view.Context) ([]view.Identity, error) {
+	if !c.tx.Opts.Auditor.IsNone() {
+		// am I the auditor?
+		if view2.GetSigService(context).IsMe(c.tx.Opts.Auditor) {
+			//
+		} else {
+			// connect to the remote auditor
+			_, err := context.RunView(newAuditingViewInitiator(c.tx))
+			if err != nil {
+				return nil, errors.WithMessagef(err, "failed requesting auditing from [%s]", c.tx.Opts.Auditor.String())
+			}
+		}
```

Comment:
if not `!view2.GetSigService(context).IsMe(c.tx.Opts.Auditor)` and get rid of the `else`?